### PR TITLE
Create elevation.html

### PR DESCRIPTION
**Pull request by jfoclpf**
as requested here: https://github.com/GeoTIFF/georaster-layer-for-leaflet/issues/145#issuecomment-2045860235

### CODE REVIEW

**Review by jfoclpf — COMMENTED** [review]
amend the extension to html

**Review by jfoclpf — COMMENTED** [review]
bug solved

**Review by jfoclpf — COMMENTED** [review]
bug solved

**Review by jfoclpf — COMMENTED** [review]
bug solved, it's hard to see like this without real time test

**Review by jfoclpf — COMMENTED** [review]
title added

**Comment by jfoclpf** [comment]
Finally, now the code is working, demo here:
https://jfoclpf.github.io/georaster-layer-for-leaflet-example/examples/elevation.html

you may approve it ;)

File: examples/elevation.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  <div id="map"></div>
+  <script>
+    var map = L.map('map')
+      
+      L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+          attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+      }).addTo(map)
+      
+      var url_to_geotiff_file = "https://contabo.joaopimentel.com/img/tif/1106.tif"
```
Comment (jfoclpf) [inline examples/elevation.html:36]: You can then change this URL by the geoTIFF file added on the PR #21

**Review by DanielJDufour — APPROVED** [review]
Awesome!

**Pull request merged**